src-vue/src/components/PullRequestsSelector.vue

```diff
@@ -0,0 +1,186 @@
+                        status="warning"
+                        :indeterminate="true"
+                        :duration="1"
+                        style="margin: 15px"
+                    />
+                </p>
+                <el-card v-else shadow="hover" v-for="pull_request in pull_requests_mods" v-bind:key="pull_request.url">
+                    <el-button type="primary" @click="installModsPR(pull_request)">Install</el-button>
+                    <a target="_blank" :href="pull_request.html_url">
+                        {{ pull_request.number }}: {{ pull_request.title }}
+                    </a>
+                </el-card>
+            </el-collapse-item>
+        </el-collapse>
+    </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue'
+import { PullRequestType } from '../../../src-tauri/bindings/PullRequestType';
+import { PullsApiResponseElement } from '../../../src-tauri/bindings/PullsApiResponseElement';
+import { invoke } from "@tauri-apps/api";
+import { ElNotification } from "element-plus";
+
+export default defineComponent({
+    name: 'PullRequestsSelector',
+    computed: {
+        pull_requests_launcher(): PullsApiResponseElement[] {
+            return this.$store.state.pull_requests_launcher;
+        },
+        pull_requests_mods(): PullsApiResponseElement[] {
+            return this.$store.state.pull_requests_mods;
+        },
+    },
+    methods: {
+        onChange(e: Object) {
+            const openedCollapseNames = Object.values(e);
+            if (openedCollapseNames.includes('1') && this.pull_requests_launcher.length === 0) {
+                this.getPullRequests('LAUNCHER');
+            }
+            if (openedCollapseNames.includes('2') && this.pull_requests_mods.length === 0) {
+                this.getPullRequests('MODS');
+            }
+        },
+        async getPullRequests(pull_request_type: PullRequestType) {
+            await invoke<PullsApiResponseElement[]>("get_pull_requests_wrapper", { installType: pull_request_type })
+                .then((message) => {
+                    switch (pull_request_type) {
+                        case "MODS":
+                            this.$store.state.pull_requests_mods = message;
+                            break;
+
+                        case "LAUNCHER":
+                            this.$store.state.pull_requests_launcher = message;
+                            break;
+
+                        default:
+                            console.error("We should never end up here");
+                    }
+                })
+                .catch((error) => {
+                    ElNotification({
+                        title: 'Error',
+                        message: error,
+                        type: 'error',
+                        position: 'bottom-right'
+                    });
+                });
+        },
+        async installLauncherPR(pull_request: PullsApiResponseElement) {
```

Comment:
I think that for more clarity, we should create a store module dedicated to PR installation with this method.

Reply:
Done in 7ca08697a75f7f7adf791b7baba09c059bd8c392